diff --git a/subsys/nrf_security/src/drivers/nrf_oberon/Kconfig b/subsys/nrf_security/src/drivers/nrf_oberon/Kconfig
--- a/subsys/nrf_security/src/drivers/nrf_oberon/Kconfig
+++ b/subsys/nrf_security/src/drivers/nrf_oberon/Kconfig
@@ -1727,7 +1727,7 @@ config PSA_NEED_OBERON_CTR_DRBG_DRIVER
 	depends on PSA_ACCEL_GET_ENTROPY
 	depends on PSA_USE_CTR_DRBG_DRIVER && !PSA_ACCEL_GENERATE_RANDOM
 	select PSA_WANT_ALG_ECB_NO_PADDING
-	select PSA_WANT_KEY_TYPE_AES
+	# select PSA_WANT_KEY_TYPE_AES

 config PSA_NEED_OBERON_HMAC_DRBG_DRIVER
 	bool
